mkosi.images/system/mkosi.postinst.chroot

```diff
@@ -83,3 +83,12 @@ if [ "$ID" = "centos" ] && [ "$VERSION" = "8" ]; then
     alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 1
     alternatives --set python3 /usr/bin/python3.9
 fi
+
+mkdir -p /usr/local/lib/sysusers.d
```

Comment:
why /usr/local instead of /usr?

Reply:
Please move it, writing /usr to /usr is fine when building an image.